install/kubernetes/helm/subcharts/mixer/templates/config.yaml

```diff
@@ -396,12 +396,12 @@ spec:
     source_workload_namespace: source.workload.namespace | "unknown"
     source_principal: source.principal | "unknown"
     source_app: source.labels["app.kubernetes.io/name"] | source.labels["app"] | "unknown"
```

Comment:
order is still bad, no? looks like there is still a bunch with the wrong order... I didn't check the rest, go through and find these... perhaps search for "source_app" and "destination_app" (snake_case, rather than camelCase) - looks like those were missed.